packages/manager/modules/overthebox/src/overthebox/details/overTheBox-details.controller.js

```diff
@@ -63,6 +63,8 @@ export default class OverTheBoxDetailsCtrl {
 
     this.guidesLink = this.OVERTHEBOX_DETAILS.guidesUrl.fr;
 
+    this.deviceIdPattern = /[a-z0-9]{8}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{12}/;
```

Comment:
```suggestion
    this.deviceIdPattern = /[a-z0-9]{8}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{12}/;
```
Should be in .constant.js